cli/defenseclaw/commands/cmd_setup.py

```
@@ -12450,3 +12450,115 @@ def _show_splunk_credentials(data_dir: str) -> None:
+        if not app.cfg.routing.version:
+            app.cfg.routing.version = "0.3.0"
+        if not app.cfg.routing.port:
+            app.cfg.routing.port = 8888
+
+        click.echo()
+        click.echo("  Setting up semantic model routing...")
+        click.echo()
+
+        # 1. Check/install vllm-sr
+        if shutil.which("vllm-sr"):
+            ver = _sp.run(["vllm-sr", "--version"], capture_output=True, text=True).stdout.strip()
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)